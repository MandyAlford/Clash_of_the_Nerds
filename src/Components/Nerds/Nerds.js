import React from 'react';
import './Nerds.scss';
import dali from '../../assets/dali.png';
import davinci from '../../assets/davinci.png';
import isaac from '../../assets/isaac.png';
import newton from '../../assets/newton.png';
import plato from '../../assets/plato.png';
import roman1 from '../../assets/roman-guy.png';
import roman2 from '../../assets/roman-guy-2.png';
import vangogh from '../../assets/vangogh.png';

const Nerds = () => {

  return (
    <section>
      <img id='dali' className='bg-img-nerd' src={ dali } />
      <img id='davinci' className='bg-img-nerd' src={ davinci } />
      <img id='isaac' className='bg-img-nerd' src={ isaac } />
      <img id='newton' className='bg-img-nerd' src={ newton } />
      <img id='plato' className='bg-img-nerd' src={ plato } />
      <img id='roman1' className='bg-img-nerd' src={ roman1 } />
      <img id='roman2' className='bg-img-nerd' src={ roman2 } />
      <img id='vangogh' className='bg-img-nerd' src={ vangogh } />
    </section>
  )
}

export default Nerds
