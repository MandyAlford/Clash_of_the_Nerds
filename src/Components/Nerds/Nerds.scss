#dali {
  position: absolute;
  height: 400px;
  top: 270px;
  left: 79%;
  z-index: 2;
  animation: slide-in-right-dali 4s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
}
@keyframes slide-in-right-dali {
  0% {
    -webkit-transform: translateX(400px);
    transform: translateX(400px);
    opacity: 0;
  }
  100% {
    -webkit-transform: translateX(0);
    transform: translateX(0);
    opacity: 1;
  }
}

#davinci {
  position: absolute;
  height: 450px;
  left: 0%;
  z-index: 2;
  top: 450px;
  animation: slide-in-left-davinci 4s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
}
@keyframes slide-in-left-davinci {
  0% {
    -webkit-transform: translateX(-1000px);
    transform: translateX(-1000px);
    opacity: 0;
  }
  100% {
    -webkit-transform: translateX(0);
    transform: translateX(0);
    opacity: 1;
  }
}

#isaac {
  position: absolute;
  height: 450px;
  left: 18%;
  z-index: 3;
  top: 450px;
  animation: slide-in-left-isaac 4s cubic-bezier(0.250, 0.460, 0.450, 0.940) 1s both;
}
@keyframes slide-in-left-isaac {
  0% {
    -webkit-transform: translateX(-1000px);
    transform: translateX(-1000px);
    opacity: 0;
  }
  100% {
    -webkit-transform: translateX(0);
    transform: translateX(0);
    opacity: 1;
  }
}

#newton {
  position: absolute;
  height: 500px;
  top: 400px;
  left: 43%;
  z-index: 3;
  animation: slide-in-right-newton 4s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
}
@keyframes slide-in-right-newton {
  0% {
    -webkit-transform: translateX(1000px);
    transform: translateX(1000px);
    opacity: 0;
  }
  100% {
    -webkit-transform: translateX(0);
    transform: translateX(0);
    opacity: 1;
  }
}

#plato {
  position: absolute;
  height: 200px;
  height: 500px;
  left: 0%;
  top: 200px;
  animation: slide-in-left-plato 4s cubic-bezier(0.250, 0.460, 0.450, 0.940) 0.5s both;
}
@keyframes slide-in-left-plato {
  0% {
    -webkit-transform: translateX(-1000px);
    transform: translateX(-1000px);
    opacity: 0;
  }
  100% {
    -webkit-transform: translateX(0);
    transform: translateX(0);
    opacity: 1;
  }
}


#roman1 {
  position: absolute;
  height: 500px;
  top: 445px;
  z-index: 4;
  left: 58%;
  animation: slide-in-right-roman1 4s cubic-bezier(0.250, 0.460, 0.450, 0.940) 1s both;
}
@keyframes slide-in-right-roman1 {
  0% {
    -webkit-transform: translateX(1000px);
    transform: translateX(1000px);
    opacity: 0;
  }
  100% {
    -webkit-transform: translateX(0);
    transform: translateX(0);
    opacity: 1;
  }
}

#roman2 {
  position: absolute;
  height: 550px;
  animation: slide-in-right-roman2 4s cubic-bezier(0.250, 0.460, 0.450, 0.940) 1s both;
  top: -110px; z-index: 1
}
@keyframes slide-in-right-roman2 {
  0% {
    -webkit-transform: translateX(1000px);
    transform: translateX(1000px);
    opacity: 0;
  }
  100% {
    -webkit-transform: translateX(0);
    transform: translateX(0);
    opacity: 1;
  }
}

#vangogh {
  position: absolute;
  height: 500px;
  top: 400px;
  z-index: 5;
  left: 82%;
  animation: slide-in-right-vangogh 4s cubic-bezier(0.250, 0.460, 0.450, 0.940) both;
}
@keyframes slide-in-right-vangogh {
  0% {
    -webkit-transform: translateX(1000px);
    transform: translateX(1000px);
    opacity: 0;
  }
  100% {
    -webkit-transform: translateX(0);
    transform: translateX(0);
    opacity: 1;
  }
}
