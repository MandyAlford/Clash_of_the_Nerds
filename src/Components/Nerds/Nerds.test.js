import React from 'react';
import { Nerds } from './Nerds';
import { shallow } from 'enzyme';

describe('Nerds', () => {
  let wrapper = shallow(<Nerds />)

  it('should match snapshot', () => {
    expect(wrapper).toMatchSnapshot()
  })

})
