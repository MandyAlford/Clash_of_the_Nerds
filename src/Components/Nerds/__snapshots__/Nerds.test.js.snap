// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Nerds should match snapshot 1`] = `
<section
  className="nerd-pile"
>
  <img
    alt="Salvador Dali"
    id="dali"
    src="dali.png"
  />
  <img
    alt="Leonardo DaVinci"
    id="davinci"
    src="davinci.png"
  />
  <img
    alt="Isaac Newton"
    id="isaac"
    src="isaac.png"
  />
  <img
    alt="Albert Einstein"
    id="newton"
    src="newton.png"
  />
  <img
    alt="Plato"
    id="plato"
    src="plato.png"
  />
  <img
    alt="Roman Man with Spear"
    id="roman1"
    src="roman-guy.png"
  />
  <img
    alt="Roman Man with Green Cape"
    id="roman2"
    src="roman-guy-2.png"
  />
  <img
    alt="Vincent Van Gogh"
    id="vangogh"
    src="vangogh.png"
  />
</section>
`;
